app/views/layouts/_user_menu.html.haml

```diff
@@ -19,6 +19,10 @@
   = link_to(edit_user_path(current_user.id)) do
     %span.fa.fa-user
     Edit Profile
+- if @conference && ((current_user && @conference.user_registered?(current_user)) || @conference.registration_open?)
+  %li
+    = link_to(conference_conference_registration_path(@conference)) do
+      = fa_icon('id-badge', text: "#{@conference.short_title} registration")
```

Comment:
I'd rather we have
```
%span.fa.fa-id-badge
      = @conference.short_title
      registration

```
like in other places

Reply:
Updated in 9b2734d8.